tests/data/power_flow/asym-line/rmatrix-with-n-xmatrix-with-n-c1-c0/input.json

```
@@ -0,0 +1,111 @@
+{
+  "version": "1.0",
+  "type": "input",
+  "is_batch": false,
+  "attributes": {},
+  "data": {
+    "node": [
+      {
+        "id": 1,
+        "u_rated": 10000.0
+      },
+      {
+        "id": 2,
+        "u_rated": 10000.0
+      },
+      {
+        "id": 3,
+        "u_rated": 10000.0
+      }
+    ],
+    "asym_line": [
+      {
+        "id": 4,
+        "from_node": 1,
+        "to_node": 2,
+        "from_status": 1,
+        "to_status": 1,
+        "r_aa": 0.6904,
+        "r_ba": 0.0495,
+        "r_bb": 0.6904,
+        "r_ca": 0.0492,
+        "r_cb": 0.0495,
+        "r_cc": 0.6904,
+        "r_na": 0.0495,
+        "r_nb": 0.0492,
+        "r_nc": 0.0495,
+        "r_nn": 0.6904,
+        "x_aa": 0.8316,
+        "x_ba": 0.7559,
+        "x_bb": 0.8316,
+        "x_ca": 0.7339,
+        "x_cb": 0.7559,
+        "x_cc": 0.8316,
+        "x_na": 0.7559,
+        "x_nb": 0.7339,
+        "x_nc": 0.7559,
+        "x_nn": 0.8316,
+        "c0" : 0.32e-9,
+        "c1" : 0.54e-9, 
+        "i_n" : 115.0
+      },
+      {
+        "id": 5,
+        "from_node": 2,
+        "to_node": 3,
+        "from_status": 1,
+        "to_status": 1,
+        "r_aa": 0.6904,
+        "r_ba": 0.0495,
+        "r_bb": 0.6904,
+        "r_ca": 0.0492,
+        "r_cb": 0.0495,
+        "r_cc": 0.6904,
+        "r_na": 0.0495,
+        "r_nb": 0.0492,
+        "r_nc": 0.0495,
+        "r_nn": 0.6904,
+        "x_aa": 0.8316,
+        "x_ba": 0.7559,
+        "x_bb": 0.8316,
+        "x_ca": 0.7339,
+        "x_cb": 0.7559,
+        "x_cc": 0.8316,
+        "x_na": 0.7559,
+        "x_nb": 0.7339,
+        "x_nc": 0.7559,
+        "x_nn": 0.8316,
+        "c0" : 0.32e-9,
+        "c1" : 0.54e-9, 
+        "i_n" : 115.0
+      }
+    ],
+    "source": [
+      {
+        "id": 6, 
+        "node": 1, 
+        "status": 1, 
+        "u_ref": 1.0,
+        "rx_ratio": 6.0
```

```suggestion
        "rx_ratio": 6.0,
        "sk": 1e15
```
To define an almost ideal swing bus